src/SFML/System/FileInputStream.cpp

```diff
@@ -156,4 +150,8 @@ std::optional<std::size_t> FileInputStream::getSize()
     return size;
 }
 
+
+////////////////////////////////////////////////////////////
+FileInputStream::FileInputStream() = default;
```

Comment:
I moved this farther down in the file to ensure the order of declarations in FileInputStream.hpp matches the order of definitions in FileInputStream.cpp. This is not necessary but it helps when flipping back and forth between both files.